pxblue/react-native-workflows
# Fix issue with EULA not showing

Fixes #81 .


#### Changes proposed in this Pull Request:
- Updated demo dependencies
- Tested a bunch of stuff to get this working
- Simplest solution was to introduce a View wrapper element around the Eula in the Self Registration Pager
- Ultimately this seems related to a bug in the react-native-pager-view. On it's own it seems fine, but there's some combination with our other dependencies that it doesn't seem to always play nice with.
- Another solution that worked was disabling the animation between successive screens, but I didn't want to have to do that and the View wrapper seems to work fine.


#### To Test:
If you want to see it broken:
- Clone the dev branch
- cd react-native-workflows/login-workflow/demo
- yarn upgrade
- cd ..
- yarn start:example

To see it fixed:
- Clone this branch
- Run the example
- Click through the self registration flow
- All of the screens should load correctly
